src/main/java/reposense/analyzer/CommitInfoAnalyzer.java

```diff
@@ -0,0 +1,75 @@
+package reposense.analyzer;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import reposense.dataobject.Author;
+import reposense.dataobject.CommitInfo;
+import reposense.dataobject.CommitResult;
+import reposense.system.LogsManager;
+
+/**
+ * Class for analyzing commit information found in the git log.
```

Comment:
I think lets standardize for all the header, 

`Analyzes commit information found in the git log.` will do.